content/docs/guides/sqlx.md

```diff
@@ -92,9 +91,11 @@ Some examples can be found [here](datasets#referencing-other-datasets).
 
 - **Post-operations**: the same as pre-operations, but defined with `post_operations { }`, and runs after the main SQL.
 
-<div className="bp3-callout bp3-icon-info-sign" markdown="1">
-  By default, pre and post operations are run in the same context as the main query for BiqQuery and SQL Data Warehouse. To disable this, see <a href="./datasets">here</a>. This feature is not supported for Redshift or Snowflake.
-</div>
+BigQuery and SQL Data Warehouse run all operations for a file (such as pre and post operations) in the same context; the executed SQL is created by joining all operations with a semi-colon `;`. For Redshift or Snowflake, operations are run as separate queries.
+
+This is useful for scripting, for example defining variables or UDFs in BigQuery before the create table statement.
+
+For Redshift or Snowflake, the `pre_operations` block is always executed as a separate query before the main block, and the `post_operations` block as a separate query afterwards.
```

Comment:
this isn't quite right, these blocks may contain multiple queries, each of which execute independently.

Reply:
Fair enough, I'll get rid of the block because I feel it doesn't add much anyway